components/mysql/actions/delete-row/delete-row.mjs

```diff
@@ -0,0 +1,48 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-delete-row",
+  name: "Delete Row",
+  description: "Delete an existing row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/delete.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async deleteRows({
+      table, condition, values = [],
+    }) {
+      const options = {
+        sql: `DELETE FROM \`${table}\` WHERE ${condition}`,
+        values,
+      };
+      return await this.mysql.executeQueryConnectionHandler(options);
```

Comment:
Could you please remove the await? Since this is an async function. [You don't need to use await to return](https://eslint.org/docs/rules/no-return-await)